src/sagemaker/tensorflow/deploying_tensorflow_serving.rst

```diff
@@ -0,0 +1,446 @@
+the S3 location defined by ``output_path``. You can call ``deploy`` on a TensorFlow
+estimator object to create a SageMaker Endpoint:
+
+.. code:: python
+
+  from sagemaker.tensorflow import TensorFlow
+
+  estimator = TensorFlow(entry_point='tf-train.py', ..., train_instance_count=1,
+                         train_instance_type='ml.c4.xlarge', framework_version='1.11')
+
+  estimator.fit(inputs)
+
+  predictor = estimator.deploy(initial_instance_count=1,
+                               instance_type='ml.c5.xlarge',
+                               endpoint_type='tensorflow-serving')
+
+
+The code block above deploys a SageMaker Endpoint with one instance of the type 'ml.c5.xlarge'.
+
+What happens when deploy is called
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Calling ``deploy`` starts the process of creating a SageMaker Endpoint. This process includes the following steps.
+
+- Starts ``initial_instance_count`` EC2 instances of the type ``instance_type``.
+- On each instance, it will do the following steps:
+
+  - start a Docker container optimized for TensorFlow Serving, see `SageMaker TensorFlow Serving containers <https://github.com/aws/sagemaker-tensorflow-serving-container>`_.
+  - start a `TensorFlow Serving` process configured to run your model.
+  - start an HTTP server that provides access to TensorFlow Server through the SageMaker InvokeEndpoint API.
+
+
+When the ``deploy`` call finishes, the created SageMaker Endpoint is ready for prediction requests. The
+`Making predictions against a SageMaker Endpoint`_ section will explain how to make prediction requests
+against the Endpoint.
+
+Deploying directly from model artifacts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you already have existing model artifacts in S3, you can skip training and deploy them directly to an endpoint:
+
+.. code:: python
+
+  from sagemaker.tensorflow.serving import Model
+
+  model = Model(model_data='s3://mybucket/model.tar.gz', role='MySageMakerRole')
+
+  predictor = model.deploy(initial_instance_count=1, instance_type='ml.c5.xlarge')
+
+Making predictions against a SageMaker Endpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you have the ``Predictor`` instance returned by ``model.deploy(...)`` or ``estimator.deploy(...)``, you
+can send prediction requests to your Endpoint.
+
+The following code shows how to make a prediction request:
+
+.. code:: python
+
+  input = {
+    'instances': [1.0, 2.0, 5.0]
+  }
+  result = predictor.predict(input)
+
+The result object will contain a Python dict like this:
+
+.. code:: python
+
+  {
+    'predictions': [3.5, 4.0, 5.5]
+  }
+
+The format of the input and the output data correspond directly to the request and response formats
+of the ``Predict`` method in the `TensorFlow Serving REST API <https://www.tensorflow.org/serving/api_rest>`_.
+
+If your SavedModel includes the right ``signature_def``, you can also make Classify or Regress requests:
```

Comment:
should "SavedModel" be monospace here?

Reply:
I think not. I Corrected the one or two places where it was to match.